blockchain/types/tx_internal_rlp_encode_test.go

```diff
@@ -1430,7 +1434,7 @@ func testTxRLPEncodeFeeDelegatedCancelWithRatio(t *testing.T) {
 		tx.TxSignatures,
 	})
 	assert.Equal(t, nil, err)
-
+	assert.Equal(t, hash(txHashRLP.Bytes()), NewTx(tx).Hash())
```

Comment:
```suggestion
	assert.Equal(t, hash(txHashRLP.Bytes()), NewTx(tx).Hash())
	assert.Equal(t, hash(senderTxHashRLP.Bytes()), NewTx(tx).SenderTxHash())
```
Maybe adding logic to check sednerTxHash value in each case?